liqwid-plutarch-extra/liqwid-plutarch-extra.cabal

```diff
@@ -12,7 +12,7 @@ maintainer:         Koz Ross <koz@mlabs.city>
 copyright:          (C) 2022 Liqwid Labs
 category:           Test
 build-type:         Simple
-tested-with:        GHC ==9.2.5
```

Comment:
I think that was an old version. I've never actually built this with 925, and I'm not sure anyone has